packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -56,114 +63,1398 @@ describe('Picker', function () {
+      expect(items[1]).toHaveTextContent('Two');
+      expect(items[2]).toHaveTextContent('Three');
+
+      expect(document.activeElement).toBe(items[2]);
+    });
+
+    it('supports controlled open state', function () {
+      let onOpenChange = jest.fn();
+      let {getAllByRole} = render(
+        <Provider theme={theme}>
+          <Picker label="Test" isOpen onOpenChange={onOpenChange}>
+            <Item>One</Item>
+            <Item>Two</Item>
+            <Item>Three</Item>
+          </Picker>
+        </Provider>
+      );
+
+      act(() => jest.runAllTimers());
+
+      expect(getAllByRole('listbox')).toHaveLength(2);
+      expect(onOpenChange).not.toBeCalled();
+
+      let picker = getAllByRole('button')[0];
+      expect(picker).toHaveAttribute('aria-expanded', 'true');
+
+      let listbox = getAllByRole('listbox')[1]; // ignore the one in the background for now
+      expect(listbox).toBeVisible();
+      expect(picker).toHaveAttribute('aria-controls', listbox.id);
+
+      let items = within(listbox).getAllByRole('option');
+      expect(items.length).toBe(3);
+      expect(items[0]).toHaveTextContent('One');
+      expect(items[1]).toHaveTextContent('Two');
+      expect(items[2]).toHaveTextContent('Three');
+
+      expect(document.activeElement).toBe(items[0]);
+    });
+
+    it('supports default open state', function () {
+      let onOpenChange = jest.fn();
+      let {getAllByRole} = render(
+        <Provider theme={theme}>
+          <Picker label="Test" defaultOpen onOpenChange={onOpenChange}>
+            <Item>One</Item>
+            <Item>Two</Item>
+            <Item>Three</Item>
+          </Picker>
+        </Provider>
+      );
+
+      act(() => jest.runAllTimers());
+
+      expect(getAllByRole('listbox')).toHaveLength(2);
+      expect(onOpenChange).not.toBeCalled();
+
+      let picker = getAllByRole('button')[0];
+      expect(picker).toHaveAttribute('aria-expanded', 'true');
+
+      let listbox = getAllByRole('listbox')[1]; // ignore the one in the background for now
+      expect(listbox).toBeVisible();
+      expect(picker).toHaveAttribute('aria-controls', listbox.id);
+
+      let items = within(listbox).getAllByRole('option');
+      expect(items.length).toBe(3);
+      expect(items[0]).toHaveTextContent('One');
+      expect(items[1]).toHaveTextContent('Two');
+      expect(items[2]).toHaveTextContent('Three');
+
+      expect(document.activeElement).toBe(items[0]);
+    });
   });
 
-  it('should have a hidden dismiss button for screen readers', function () {
-    let onOpenChange = jest.fn();
-    let {getByRole, getAllByRole} = render(
-      <Provider theme={theme}>
-        <Picker label="Test" onSelectionChange={onSelectionChange} onOpenChange={onOpenChange}>
-          <Item>One</Item>
-          <Item>Two</Item>
-          <Item>Three</Item>
-        </Picker>
-      </Provider>
-    );
+  describe('closing', function () {
+    it('can be closed by clicking on the button', function () {
```

Comment:
same question about the closed tests and refactoring the common parts into shared code.